calm/draft/2024-03/meta/core.json

```diff
@@ -0,0 +1,229 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/finos-labs/architecture-as-code/main/calm/draft/2024-03/meta/core.json",
+  "title": "Common Architecture Language Model (CALM) Vocab",
+  "properties": {
+    "nodes": {
+      "type": "array",
+      "items": {
+        "$ref": "#/defs/node"
+      }
+    },
+    "relationships": {
+      "type": "array",
+      "items": {
+        "$ref": "#/defs/relationship"
+      }
+    },
+    "required": [
+      "nodes",
+      "relationships"
+    ]
+  },
+  "defs": {
+    "node": {
+      "type": "object",
+      "properties": {
+        "uniqueId": {
```

Comment:
I noticed that we have a mix of camel case and kebab case in the spec. Most are kebab case but fields like `uniqueId` and `dataClassification` are camel. If there isn't a reason behind this, maybe we could take this opportunity to make them consistent.

Reply:
Let's raise that as a separate issue given the amount of changes we've already asked for, but agree with making this happen.

Reply:
@aidanm3341 agree this should be a separate issue and would be good to do now we're into a next version of the schema.